hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -1072,15 +1072,14 @@
 
             Check the peripheral is reset via timeout accessing their CSRs while the corresponding
             reset is active. The per peripheral reset controls in increasing order by
```

Comment:
I think this description needs an update too. The original testing method was a lot simpler - program the watchdog timer,  just do a simple read while the reset is held (which should hang), then the chip should reset due to watchdog expiration. 

You are instead writing a known data to a selected register before reset, and reading it back after reset to confirm the peripheral did reset (which is totally fine too). 

I think you might have done it this way because `dif_rstmgr_software_reset()` pulses the SW reset. @timothytrippel, it may be worth adding an arg to this DIF to leave the reset held.

Reply:
updated